### ISSUE TITLE
[BUG] Display bug after CPU cores reach high temperatures

### ISSUE THREAD

**Issue by turkeyphant** [aristocratos/bashtop#103]
**Describe the bug**

After any thread goes over ~75C, the temperature units column has a display bug:

![image](https://user-images.githubusercontent.com/8129583/81096277-0a11d680-8efe-11ea-8ab7-98efccf56489.png)

**To Reproduce**

Stress test CPU until required temperature reached.

**Expected behavior**

Temperature units displayed correctly:

![image](https://user-images.githubusercontent.com/8129583/81096352-20b82d80-8efe-11ea-88f7-740a00f8ed3b.png)

**Info (please complete the following information):**
 - Bashtop version: 0.8.18
 - Linux distribution and version: Linux 4.19.107-Unraid.
 - Terminal used: webterminal
 - Font used: webterminal
 - Bash version, "bash --version" (version 4.4 or above is required): 5.0.11(1)-release (x86_64-slackware-linux-gnu)
 - Locales: output of "locale -v"
```
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE=C
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

**Additional context**

contents of `$HOME/.config/bashtop/error.log`

```
cat $HOME/.config/bashtop/error.log
 
New instance of bashtop version: 0.8.18 Pid: 26206
 
New instance of bashtop version: 0.8.18 Pid: 26365
 
New instance of bashtop version: 0.8.18 Pid: 24688
 
New instance of bashtop version: 0.8.18 Pid: 27063
 
New instance of bashtop version: 0.8.18 Pid: 20379
 
New instance of bashtop version: 0.8.18 Pid: 16378

```

**Comment by aristocratos** [comment]
Yeah, the error is because I missed a check if current temperature is higher than the set cpu high value which it uses to calculates the graph. Will fix in new version that will be pushed later today.

**Comment by aristocratos** [comment]
Turns out that wasn't the error. But a error in the graph creation when value was above 100%.
Fixed in v0.8.25

**Comment by turkeyphant** [comment]
Now it seems to completely ignore temperatures sometimes (v0.8.28):

![image](https://user-images.githubusercontent.com/8129583/81514973-c407a900-9329-11ea-8b1b-19590c5d14c1.png)

**Comment by aristocratos** [comment]
@turkeyphant  New fix in v0.8.29
